doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -38,7 +38,12 @@
 <a name="4.5"></a>
 ## 4.5 CNI plugins
 
-> This chapter should describe which CNI plugins are part of the Rerefence Architecture.
+The used CNI multiplexer/metapulgin must be [DANM](https://github.com/nokia/danm)
+as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).
+ [Calico](https://github.com/projectcalico/cni-plugin) must be used as the CNI what complies with the basic networking assumptions of Kubernetes based on the requirement `req.inf.ntw.02`.
+For the network of signalling connections the built in IPVLAN CNI of DANM must be used as it provides NAT-less connectivity (`req.inf.ntw.03`). For the user plane network(s) fullfilling requirement `req.inf.ntw.04` the [User Space CNI](https://github.com/intel/userspace-cni-network-plugin) must be used.
```

Comment:
@CsatariGergely thanks for this - it's helpful. What would make this complete, for me, is for the reasons that the alternatives (i.e. Multus, Flannel, etc.) are able to meet the requirements, as that is the unanswered question left open here.

Reply:
@tomkivlin is there anything still open here?

Reply:
For Multus v DANM, yes. For the Calico part, I don't think it's as simple-a-comparison. For example, Flannel and Calico together (/canal) could deliver the requirements, yes?  And there are other CNI plugins that also support NetworkPolicy such as Cilium and Weave Net.

Reply:
TBH I don't think Canal is actively maintained by anyone, or is widely used
Agree Cilium could be an interesting alternative, however does its NP implementation work if your traffic is not in L7?